### PR TITLE
Fix install jira-python

### PR DESCRIPTION
**Pull request by davies**
jira-pytyhon package should be installed by

  sudo pip install jira

cc @pwendell

### CODE REVIEW

File: dev/create-release/releaseutils.py [hunk 1]
```diff
@@ -27,7 +27,7 @@
     from jira.exceptions import JIRAError
 except ImportError:
     print "This tool requires the jira-python library"
```
Comment (JoshRosen) [inline dev/create-release/releaseutils.py]: Replace `jira-python` here, too?

**Comment by JoshRosen** [comment]
Switching from `jira-python` to `jira` looks good to me; we made a similar change for spark-prs: https://github.com/databricks/spark-pr-dashboard/issues/52

File: dev/create-release/releaseutils.py (hunk 1 above)
Comment (davies) [inline dev/create-release/releaseutils.py]: This library is called 'jira-python', but is indexed as 'jira' in pip.

**Comment by pwendell** [comment]
LGTM

**Pull request closed without merging**